lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -31,19 +31,14 @@ def has_permission(user, permission, course_id=None):
 CONDITIONS = ['is_open', 'is_author', 'is_question_author', 'is_team_member_if_applicable']
 
 
+@request_cached
 def get_team(commentable_id):
     """ Returns the team that the commentable_id belongs to if it exists. Returns None otherwise. """
```

Comment:
Heh. Now that I look at this code, I suspect this is the extra query that's now happening ,or possibly the query. If it was getting cached before whether it was None or not, my initial implementation would have not cached it, and caused redundant calls.

Gonna test this right now.